SFDO-Tooling/Metecho
# Fix Nested Accordion Layout [nested-changes]

Layout would break when expanding ignored changes accordion in the retrieve changes modal. 

Before and After
![me-changes-modal](https://user-images.githubusercontent.com/1581694/169411668-09ed6c82-899c-4ca8-84bd-103394f3aee5.jpg)

To test: 

- Make a dev org on a task or use existing one if you have one
- Make some changes to the org (see pinned instructions if you don't know how in Slack). 
- Open the "Retrieve Changes from Dev Org" modal
- Leave the first screen on the default selection and choose "Save & Next"
- Then select a checkbox from the All changes list and choose "Ignore Selected Changes"
- Expand the Ignored changes if they weren't already and see if the spacing looks correct.
